pkg/runner/runner.go

```diff
@@ -24,24 +32,33 @@ func NewRunner() (*JMeterRunner, error) {
 		return nil, fmt.Errorf("could not initialize JMeter runner variables: %w", err)
 	}
 
-	return &JMeterRunner{
+	r := &JMeterRunner{
```

Comment:
Can we close this in some library and don't leak cloud implementation details into the executor please.

Reply:
Maybe some convention will be enough here for all possible executors who needs grpc connection